pages/ecosystem.vue

```diff
@@ -136,177 +137,141 @@ useHead({
+
+  return [];
+}
+
+function getTierDescription(tierName: string): string {
+  const tier = tiers.find((tier: any) => tier.name === tierName);
+  return tier!.description || "";
+}
+
+function updateTierFilter(filterValue: string, isChecked: boolean) {
+  isChecked
+    ? tierFilters.value.push(filterValue)
+    : (tierFilters.value = tierFilters.value.filter(
+        (tier: any) => tier !== filterValue
+      ));
+}
+
+function updateTierFilters(newTierFilters: string) {
+  const newTierFiltersAsArray = newTierFilters.split(",");
+  tierFilters.value = newTierFiltersAsArray;
+}
+
+const isTierFilterChecked = (filterValue: string): boolean =>
+  tierFilters.value.includes(filterValue);
+
+const joinAction: GeneralLink = {
+  url: "https://github.com/qiskit-community/ecosystem#ecosystem--",
+  label: "Join the ecosystem",
+};
 
 // TODO: Refactor tracking
 // export default class EcosystemPage {
 //   routeName: string = 'ecosystem';
-
-//   joinAction: GeneralLink = {
-//     url: 'https://github.com/qiskit-community/ecosystem#ecosystem--',
-//     label: 'Join the ecosystem'
-//   };
 // }
 </script>
 
-<style lang="scss">
-// TODO: Review old CSS
-// .ecosystem-header {
-//   &__hero {
-//     .cds--col-max-8 {
-//       max-width: 100%;
-//       flex: 0 0 100%;
-//     }
-//   }
-
-//   &__cta {
-//     height: 50%;
-//   }
-// }
-
-// .ecosystem__filters-result-section {
-//   margin-top: 4rem;
-
-//   &__tiers {
-//     text-transform: capitalize;
-//   }
-// }
-
-// .cds--col-sm-4 {
-//   padding-bottom: carbon.$spacing-08;
-// }
-
-// .app-card {
-//   &__description {
-//     .cds--row {
-//       margin-left: 0;
-//     }
-//   }
-
-//   &__tags {
-//     flex-direction: row;
-//   }
-// }
-
-// .bx--accordion__title {
-//   display: flex;
-//   flex-direction: row;
-//   gap: 10px;
-
-//   p,
-//   .p {
-//     margin-bottom: 0;
-//   }
-// }
-
-// .bx--accordion__item {
-//   background-color:carbon.$cool-gray-20;
-//   border-bottom: none;
-// }
-
-// .bx--accordion__item--active .bx--accordion__content {
-//   padding: 0;
-//   margin: 0;
-//   padding-top: 0;
-//   padding-bottom: 0;
-// }
-
-// .bx--accordion__item:last-child {
-//   border-bottom: none;
-// }
-
-// .project-card {
-//   &__license {
-//     font-size: 12px;
-//     margin-right: carbon.$spacing-05;
-//     margin-top: calc($spacing-01 / 2);
-//   }
-
-//   &__star {
-//     display: flex;
-//     flex-direction: row;
-
-//     svg {
-//       margin-top: calc($spacing-01 / 2);
-//       margin-right: carbon.$spacing-01;
-//       fill:carbon.$cool-gray-60;
-//     }
-
-//   }
-
-//   .app-card__title {
-//     font-size: 20px;
-//   }
-// }
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+.ecosystem__filters-result-section {
+  margin-top: 4rem;
+}
+
+.cds--col-sm-4 {
+  padding-bottom: carbon.$spacing-08;
+}
+
+.app-card {
+  &__description {
+    .cds--row {
+      margin-left: 0;
+    }
+  }
+
+  &__tags {
+    flex-direction: row;
+  }
+}
```

Comment:
Should this code be on the `AppCard` component?

Reply:
It's the current implementation and it's needed to avoid misalignments in the cards. But I agree that a cleaner implementation might be possible.